### DISCUSSION TITLE
关于预览

### DISCUSSION THREAD

**Discussion by ncwzdlsd** [walinejs/waline#1693] [Q&A] [answered]
在通过.vercel.app预览时，能否显示表情等图标

**Comment by lizheming** [comment, marked as answer]
那只是一个 Debug 页面，暂时不支持用户去配置内容。如果真的需要的话，可以自己 Vercel rewrite `/` 路由到自己的静态页面渲染一个 Waline，就支持自定义配置了。https://vercel.com/docs/project-configuration#project-configuration/rewrites
